src/SprykerEco/Yves/Adyen/Theme/default/credit_card.twig

```diff
@@ -0,0 +1,79 @@
+<script type="text/javascript" src="https://checkoutshopper-test.adyen.com/checkoutshopper/assets/js/sdk/checkoutSecuredFields.1.1.1.min.js"></script>
+<div class="cards-div">
+    <div class="js-chckt-pm__pm-holder">
+        <input type="hidden" name="txvariant" value="card" />
+        <br />
+        <label>
+            <span class="input-field" data-hosted-id="hostedCardNumberField" data-cse="encryptedCardNumber" />
+        </label>
+        <br />
+        <label>
+            <span class="input-field" data-hosted-id="hostedExpiryDateField" data-cse="encryptedExpiryDate" />
+        </label>
+        <br />
+        <label>
+            <span class="input-field" data-hosted-id="hostedSecurityCodeField" data-cse="encryptedSecurityCode" />
+        </label>
+    </div>
+</div>
+
+<script type="text/javascript">
+
+    // Define a custom style.
+    var styleObject = {
+        base: {
+            color: 'black',
+            fontSize: '16px',
+            fontSmoothing: 'antialiased',
+            fontFamily: 'Helvetica'
+        },
+        error: {
+            color: 'red'
+        },
+        placeholder: {
+            color: '#d8d8d8'
+        },
+        validated: {
+            color: 'green'
+        }
+    };
+
+    // Load Secured Fields.
+    var securedFields = csf({
+        configObject : {
+            originKey : "pub.v2.8215323288033856.aHR0cDovL3d3dy5kZS5hZHllbi5sb2NhbA.wqS4CdQ29skrG79bxc92KusqP7acm6FNEXeWj9IKO0o"
+        },
+        rootNode: '.cards-div',
+        paymentMethods : {
+            card : {
+                sfStyles : styleObject,
+                placeholders: {
+                    hostedCardNumberField : '4111 1111 1111 1111',
+                    hostedExpiryDateField : '08/18',
+                    hostedSecurityCodeField : '737'
+                }
+            }
+        }
+    });
+
+    // Listen to events.
+    securedFields.onLoad(function(){
+        console.log('All fields have been loaded');
+    });
+
+    // Triggers when all credit card input fields are valid - and triggers again if this state changes.
+    securedFields.onAllValid(function(allValidObject){
+        if(allValidObject.allValid === true){
+
+            alert('All credit card input is valid :-)');
```

Comment:
This will not be in the final version, will it?

Reply:
It's a draft version. Final version will be implemented by frontend team.